frontend/src/metabase/components/Modal.jsx

```diff
@@ -3,7 +3,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 
-import { CSSTransitionGroup } from "react-transition-group";
+import { CSSTransition, TransitionGroup } from "react-transition-group";
```

Comment:
Old `react-transition-group` (v1) uses old React lifecycle hook names, so we need to upgrade it. The migration are performed following [this guide ](https://github.com/reactjs/react-transition-group/blob/master/Migration.md)